spring-kafka/src/main/java/org/springframework/kafka/event/ConsumerPartitionPausedEvent.java

```diff
@@ -46,14 +47,24 @@ public ConsumerPartitionPausedEvent(Object source, Object container, TopicPartit
 	/**
 	 * Return the paused partition.
 	 * @return the partition.
+	 * @deprecated replaced by {@link #getPartition()}
 	 */
+	@Deprecated(since = "3.3", forRemoval = true)
 	public TopicPartition getPartitions() {
 		return this.partition;
 	}
 
+	/**
+	 * Return the paused partition.
+	 * @return the partition.
+	 */
```

Comment:
Need the `@since` tag here.

Reply:
Oh Thanks for letting me know. I've added it.